integrations/ScriptLoader.js

```diff
@@ -4,15 +4,17 @@ import logger from "../utils/logUtil";
 const defaultAsyncState = true;
 
 export const LOAD_ORIGIN = "RS_JS_SDK";
+export const defaultNonNativeSDKState = "false";
 
-const ScriptLoader = (id, src, async = defaultAsyncState) => {
+const ScriptLoader = (id, src, async = defaultAsyncState, isNonNativeSDK = defaultNonNativeSDKState) => {
   logger.debug(`in script loader=== ${id}`);
   const js = document.createElement("script");
   js.src = src;
   js.async = async === undefined ? defaultAsyncState : async;
   js.type = "text/javascript";
   js.id = id;
   js.dataset.loader = LOAD_ORIGIN;
+  js.dataset.isNonNativeSDK = isNonNativeSDK;
```

Comment:
Set this field only if the user provides the value. No default.

Reply:
Addressed